lib/sorbet-rails/config.rb

```diff
@@ -46,8 +46,10 @@ def initialize
         :active_record_assoc,
         :active_record_finder_methods,
         :custom_finder_methods,
-        :enumerable_collections,
+        :enumerable_collections
       ]
+
+      @enabled_model_plugins << :active_storage_methods if defined?(T.unsafe(ActiveStorage))
```

Comment:
Hmm if we only support rails v6.0, we might need to change this condition.

Reply:
This doesn't really cause any problems since the plugin itself checks whether the given model has any `attachment_reflections`